container/snippets/quickstart.js

```diff
@@ -0,0 +1,41 @@
+// Copyright 2017 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+// [START gke_list_cluster]
+async function main() {
+  // [START container_quickstart]
```

Comment:
AFAICT, this region tag is **only** used [as a standalone sample](https://cloud.google.com/kubernetes-engine/docs/samples/container-quickstart).

I'm not sure if those are in-scope for these PRs. **If they are**, feel free to remove the _Do not merge_ label.